python/az/aro/azext_aro/_params.py

```diff
@@ -54,10 +57,25 @@ def load_arguments(self, _):
         c.argument('service_cidr',
                    help='CIDR of service network. Must be a minimum of /18 or larger.',
                    validator=validate_cidr('service_cidr'))
+        c.argument('software_defined_network', arg_type=get_enum_type(['OVNKubernetes', 'OpenShiftSDN']),
+                   options_list=['--sdn-type'],
+                   help='SDN type either "OVNKubernetes" or "OpenShiftSDN (default)"',
```

Comment:
Lets reverse this order. Default first.

Reply:
Done for SDN and host encryption params.